internal/cli/kraft/cloud/instance/list/list.go

```diff
@@ -68,10 +69,19 @@ func (opts *ListOptions) Run(ctx context.Context, args []string) error {
 		kraftcloud.WithToken(config.GetKraftCloudTokenAuthConfig(*auth)),
 	)
 
-	instances, err := client.WithMetro(opts.metro).List(ctx)
+	instListResp, err := client.WithMetro(opts.metro).List(ctx)
 	if err != nil {
 		return fmt.Errorf("could not list instances: %w", err)
 	}
 
-	return utils.PrintInstances(ctx, opts.Output, instances...)
+	var is []kcinstances.GetResponseItem
+	for _, instItem := range instListResp {
+		i, err := client.WithMetro(opts.metro).GetByUUID(ctx, instItem.UUID)
+		if err != nil {
+			return fmt.Errorf("gettings details of instance %s: %w", instItem.UUID, err)
```

Comment:
```suggestion
			return fmt.Errorf("getting details of instance %s: %w", instItem.UUID, err)
```

No typo is safe

Reply:
Laser eyes! Done in [da0715e8..98db95e3](https://github.com/unikraft/kraftkit/compare/da0715e8aa67c125930667f42ef9633f0cb295ed..98db95e38fed65aa0fbfbaf7fddd20f5f08d35cd)